.gitignore

```diff
@@ -3,6 +3,7 @@
 __pycache__
 *.qmlc
 *.qmlc.*
+.ztr-directory
```

Comment:
The `.ztr-directory` folder is not specific to this repository. It may be specific to some of your tooling, but then you're better off adding this gitignore to your personal .gitignore file.

To create a personal .gitignore file, use `git config --global core.excludesfile /home/5axes/.gitignore` (or wherever you want to store your .gitignore file). Then add this line to that .gitignore file.